### DISCUSSION TITLE
Restart / reload button

### DISCUSSION THREAD

**Discussion by charlesyu08** [EDCD/EDMarketConnector#1159] [Ideas]
A restart / reload (option / button) to reload the plugins

**Comment by Athanasius** [comment]
There is some work in progress on a new plugin framework (which will support the current paradigm too, no big flag day for third-party plugin devs).  Until that's complete, merged, and released unfortunately we can't support reloading of plugins due to how the current code works.

**Comment by Athanasius** [comment]
Having said that ... may I ask why you feel a need for reload/restarting a plugin?

i.e. are you a plugin developer who wants be able to more quickly test your own code, without needing to restart the whole program?  Or do you have a reason as a user to use such a feature?

**Reply by charlesyu08** [reply]
I am a new user of the application, it would be great to have such features as u might not need all the plugins all the time, for example, u might only want certain plugin for exploration, and other plugins while doing missions, etc
